config/sync/update.settings.yml

```diff
@@ -0,0 +1,12 @@
+_core:
```

Comment:
Is this config file for the upgrade status module which is only enabled on local? Probably will want to remove this.